src/transformers/models/oneformer/backbone_swin_oneformer.py

```diff
@@ -0,0 +1,788 @@
+# coding=utf-8
```

Comment:
Same comment here, no need to implement the backbone entirely in a new file, but rather make use of the `AutoBackbone` class.